src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -87,6 +99,39 @@ export const OAuthManagement: React.FunctionComponent = () => {
           isShowingConfirmModal={isShowingConfirmModal}
         />
       )}
+      <Modal
+        show={isShowingVerification}
+        animation={false}
+        backdrop="static"
+        onHide={() => setIsShowingVerification(false)}
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>
+            <Typography variant="headline1">Submit Verification</Typography>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            In order to verify an OAuth client please send an email to{' '}
+            <a href={'mailto:synapseinfo@sagebase.org'}>
+              synapseinfo@sagebase.org
+            </a>
+            .
+          </p>
+          <b>Please list the following items in your email:</b>
+          <ul>
+            <li>Your name</li>
+            <li>
+              The ID of the client to be verified <br />
+              <i>(You can find this within Actions)</i>
+            </li>
+            <li>A description of your application</li>
+          </ul>
+          <p>
+            Verification can take up to X weeks and we will notify you via X.
```

Comment:
The design does not indicate a specific time.

Reply:
Please comment this out until we can establish the agreement.